connectors/v2/deezer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#page_sidebar';
+
+Connector.getArtist = function() {
+	var artists = $.map($('.player-track-artist').children(), function(artist) {
+		return $.text(artist);
+	});
+	return artists.join(', ');
```

Comment:
This doesn't look like it makes sense to me? Are things really that wacky on the page?